manifests/03-role-bindings.yaml

```diff
@@ -11,3 +11,31 @@ roleRef:
   kind: ClusterRole
   name: elasticsearch-operator
   apiGroup: rbac.authorization.k8s.io
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
I think we would want the EO to create these instead.. the manifest files should be more about configuring the EO SA which will translate to what OLM will do for us

Reply:
done